src/pages/login/login.ts

```diff
@@ -11,5 +11,12 @@ export class LoginPage {
   title: string = 'Login';
   journalPage: Page = JournalPage;
 
-  constructor(public navCtrl: NavController) {}
+  constructor(public navCtrl: NavController, public viewCtrl: ViewController) { }
+
+  navToJournal() {
+    // Nav to Journal and they remove login from the stack so we can pop to root later
+    this.navCtrl
+      .push(JournalPage)
```

Comment:
shouldn't we just root pop to 1st view controller (as 0 is login page)?

Reply:
pushing makes that we still have old vcs in stack

Reply:
Not sure what you mean @zielinskikamil, did you have a method in mind?
This will remove the login page from the stack.

Reply:
nvm, I just realised that this code is fine :)